eth2deposit/key_handling/keystore.py

```diff
@@ -85,11 +91,18 @@ def from_json(cls, path: str) -> 'Keystore':
         with open(path, 'r') as f:
             json_dict = json.load(f)
         crypto = KeystoreCrypto.from_json(json_dict['crypto'])
-        pubkey = json_dict['pubkey']
         path = json_dict['path']
         uuid = json_dict['uuid']
         version = json_dict['version']
-        return cls(crypto=crypto, pubkey=pubkey, path=path, uuid=uuid, version=version)
+        description = json_dict.get('description', '')
+        pubkey = json_dict.get('pubkey', '')
+        return cls(crypto=crypto, description=description, pubkey=pubkey, path=path, uuid=uuid, version=version)
+
+    @staticmethod
+    def _process_password(password: str) -> bytes:
+        password = normalize('NFKD', password)
+        password = ''.join(c for c in password if ord(c) not in UNICODE_CONTROL_CHARS)
```

Comment:
Maybe add an edge test case of control characters, e.g., _process_password('\t')?